test/setup/snapshot-resolver.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'path';
+
+export default {
```

Comment:
```suggestion

// We need this because we run our tests for different react versions.
// Each version render components differently thus returning a different
// snapshot. We'll store each snapshot in different folder.
// __react_16_snapshots__, __react_17_snapshots__ and __react_18_snapshots__
export default {
```